docs/release-notes/cloud/cloud-2023-01.md

```diff
@@ -0,0 +1,88 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud January 2023.
+included_jira_versions: ['2023.01']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/6.7.36
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/6.7.76
+---
+
+# Cloud January 2023
+
+These release notes are for the Codacy Cloud updates during January 2023.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   The Codacy Coverage Reporter now supports [automatic commit SHA-1 detection](../../coverage-reporter/troubleshooting-coverage-cli-issues.md#commit-detection) on Argo CD. (TS-140)
+-   Extended the visibility of the [coding standards page](../../organizations/using-a-coding-standard.md) to all users, regardless of their permission level. (IO-359)
+-   The pull request detail page now displays the [commits that are missing coverage data](../../coverage-reporter/index.md#validating-coverage) for Codacy to calculate the coverage metrics. This information is also visible in the analysis logs. (IO-152)
+
+    ![Pull request analysis logs showing the commits that are missing coverage data](../images/io-152b.png)
+
+    <!--NOTE This Jira key appears on two release note pages because the feature had an update after being released (see https://codacy.atlassian.net/browse/COV-137)-->
+
+-   Codacy now supports [reporting a summary of the coverage metrics](../../repositories-configure/integrations/github-integration.md#enabling) directly on your pull requests. For now, this feature is only supported on GitHub Cloud. (COV-2, COV-3)
+
+    ![Coverage summary on GitHub](../images/cov-2.png)
+
+## Bug fixes
+
+-   Codacy no longer counts ignored files towards the [maximum number of files for calculating complexity](../../faq/code-analysis/does-codacy-place-limits-on-the-code-analysis.md). (TS-93)
+-   Fixed an issue that caused all ESLint 8 code patterns to be turned off by default when adding a new repository. (TS-50)
+-   Fixed the repository page not opening for Bitbucket repositories with many branch restrictions and repository:admin users. (PLUTO-290)
```

Comment:
@claudiacarpinteiro please push a commit to this pull request branch with any edits that you decide to do on this release note. :rocket:

Reply:
I pushed another commit just to add the "code" style to the `repository:admin` scope. :nerd_face: